### ISSUE TITLE
How to get the .npy file?

### ISSUE THREAD

**Issue by muximuxi** [autonomousvision/factor-fields#4]
If I use my own data  in SDF , how to load it and get the .npy file?

**Comment by muximuxi** [comment]
and by the way, how to get the transforms_test.json ,transforms_train.json and transforms_val.json  like demo data in nerf_synthetic? thank you

**Comment by apchenstu** [comment]
> If I use my own data in SDF , how to load it and get the .npy file?
good point;) I have uploaded the file (i.e., "mesh2SDF_data_process.py") to the script folder.

**Comment by apchenstu** [comment]
> 
please refer to here if you want to process your own data, the format is the same as the instant-ngp
https://github.com/NVlabs/instant-ngp/blob/master/docs/nerf_dataset_tips.md
